tests/test_cms_menus.py

```diff
@@ -404,6 +405,239 @@ def test_menu_without_softroots(self):
 
         self.assertTreeQuality(hard_root, mock_tree, 'level', 'title')
 
+    def test_menu_with_node_hidden(self):
+        """
+        Tree in fixture :
+               root
+                   aaa
+                       aaa1( Hide_node = True) hidden node
+                           ccc
+                               ddd
+                       aaa2
+                   bbb
+        tag: show_menu 0 100 0 100
+        expected result 1:
```

Comment:
I don't understand why this is the expected result. Please add more detail to the description to help me understand what we are trying to test here. If I had to repair this in the future I would have no idea.

Reply:
Updated docstrings for the tests